metaci/build/models.py

```diff
@@ -713,6 +713,19 @@ def _get_flow_options(self) -> dict:
                 "sandbox_date": self.build.release.sandbox_push_date,
                 "production_date": self.build.release.production_push_date,
             }
+        if (
+            self.build.plan.role == "push_sandbox" and self.build.release
+        ):  # override lives in MetaCI
+            options["push_sandbox"] = {
+                "version": f"{self.build.release.version_number}",
+            }
+        if (
+            self.build.plan.role == "push_production" and self.build.release
+        ):  # override lives in MetaCI
+            options["push_all"] = {
```

Comment:
@davisagli is it ok that we're using push_all? Theres no push_production task.

Reply:
Yeah, I think push_all is what we want, because on production Tuesday we want to include any new sandboxes that were created with the old version since sandbox Wednesday.